docs/usage/cli.md

```diff
@@ -171,6 +179,5 @@ $ babel-node --debug --presets es2015 -- script.js --debug
 | `-p, --print`            |                      | Evaluate script and print result |
 | `-i, --ignore [regex]`   | `node_modules`       | Ignore all files that match this regex when using the require hook |
 | `-x, --extensions`       | `".js",".jsx",".es6",".es"` | List of extensions to hook into |
-| `--no-babelrc`           |                      | Ignore config defined in .babelrc/package.json |
```

Comment:
https://github.com/babel/babel.github.io/pull/959#issuecomment-258958497 - there isn't a `--no-babelrc` option for babel-node - it's for babel-cli